src/main/reactjs/src/components/clerkTranslator/overview/AuthorisationDetails.tsx

```diff
@@ -67,7 +71,22 @@ const AuthorisationsListing = ({
               </Text>
             </TableCell>
             <TableCell>
-              <Text>{a.basis}</Text>
+              <div className="columns gapped-xs">
+                <Text>{a.basis}</Text>
+                {a.basis === AuthorisationBasisEnum.AUT && (
+                  <Tooltip
+                    title={`${t(
+                      'fields.autDate'
+                    )}: ${DateUtils.formatOptionalDate(a.autDate)}`}
```

Comment:
Onko mahdollista että peruste on `AUT` mutta tutkintopäivä puuttuu?

Reply:
pitäs olla mukana, mutta parempi laittaa - kun se jostain syystä ei löydy.

Reply:
Ai joo, näinhän tuo formatOptionalDate tosiaan tekikin. Hyvä näin siis.

Reply:
Kannan tasolla ei pakoteta tutkintopäivämäärän olemassaoloa AUT-perusteisille auktorisoinneille. Voi olla toki hyvä uusille luotaville auktorisoinneille se frontin päästä pakottaa.

Reply:
Merkkasin unresolved jotta viesti näkyy

Reply:
juu, formatOptionalDate laittaa kuitenkin viivan kun se ei ole olemassa muistaakseni

Reply:
Joo, en tosiaan ottanut UI:hin kantaa vaan noin muuten huomiona